arangod/Aql/AqlFunctionFeature.cpp

```diff
@@ -216,7 +216,7 @@ void AqlFunctionFeature::addNumericFunctions() {
   // numeric functions
   add({"FLOOR", ".", flags, &Functions::Floor});
   add({"CEIL", ".", flags, &Functions::Ceil});
-  add({"ROUND", ".", flags, &Functions::Round});
+  add({"ROUND", ".|.,.", flags, &Functions::Round});
```

Comment:
AFAIK, `|` isn't a logical OR but a separator like `,` that also indicates the beginning of optional arguments.

```suggestion
  add({"ROUND", ".|.", flags, &Functions::Round});
```

Reply:
@Simran-B : true, the signature should be changed according to your suggestion.

Reply:
Ok good to know... Make sense now 😃